docs/framework/install/troubleshoot-blocked-installations-and-uninstallations.md

```diff
@@ -12,6 +12,9 @@ ms.assetid: c3fdfbc1-ed99-4202-a2b0-8c4f1646385d
 
 When you run the [web or offline installer](guide-for-developers.md) for the .NET Framework 4.5 or later versions, you might encounter an issue that prevents or blocks the installation of the .NET Framework. The following table lists possible blocking issues and provides links to troubleshooting information.
 
+> [!NOTE]
+> To install or uninstall current versions of .NET (not .NET Framework), visit [the .NET site](https://dotnet.microsoft.com/download) to download the latest versions. For uninstalling newer versions of .NET, see [.NET uninstall tool](https://learn.microsoft.com/dotnet/core/additional-tools/uninstall-tool-overview).
```

Comment:
The link to `uninstall-tool-overview` should be relative to the markdown file in the repo, not the published page.